package.json

```diff
@@ -92,19 +85,24 @@
     "webpack-dashboard": "^0.2.0",
     "webpack-dev-server": "^1.16.2"
   },
-  "babel": {
-    "env": {
-      "test": {
-        "plugins": [
-          [
-            "__coverage__",
-            {
-              "ignore": "**/*-spec.js"
-            }
-          ]
-        ]
-      }
+  "jest": {
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
```

Comment:
I couldn't find such a proxy, but there's an example here: https://facebook.github.io/jest/docs/tutorial-webpack.html that would do what we want, I think.  Search for `fileTransformer` there.